app/src/main/res/layout/fragment_blinds_settings.xml

```diff
@@ -11,4 +12,32 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content" />
 
+
+    <!-- add blinds button -->
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/add_btn"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentBottom="true"
+        android:layout_gravity="bottom|right"
+        android:clickable="true"
+        android:focusable="true"
+        android:layout_marginEnd="16dp"
+        android:layout_marginBottom="16dp"
+        app:srcCompat="@drawable/ic_add_24" />
+
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
```

Comment:
Android studio gives a warning about setting a content description. Please do this, or supress the warning if you think it is unnecessary. Please do not ignore warnings.